packages/core/types/index.d.ts

```diff
@@ -2248,7 +2248,7 @@ export namespace Asciidoctor {
      * Create a new Inline element.
      * @returns a new Inline element
      */
-    static create(parent: AbstractBlock, context: string, text?: string, opts?: any): Inline;
+    static create(parent: AbstractBlock, context: string, text?: string, opts?: Record<string, unknown>): Inline;
```

Comment:
:thinking: I'm trying to think if it's possible to use something other than `string` as key... but I think the answer is no.

Reply:
Could there anything besides a `string` as value?

Reply:
No I think it would be fine.
In Ruby, you can have numeric keys but that's not possible in JavaScript so `Record<string, unknown>` should be OK.

```rb
h = {1 => 'one', 2 => 'two', 3 => 'three'}
```